guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>In <u>the </u>case of the machine learning model, <s>similarly</s> the data prepared by the <code>prepare_tabulardata</code> method is ready to be passed to the <code>MLModel</code> method, along with the selected machine learning model for training. Here, for demonstration purposes, the Gradient BoostingRegressor model from scikit-learn is used, which is passed into the <code>MLModel</code> function, along with its parameters. <s>First, the </s><code><s>MLModel</s></code><s> framework is imported from </s><code><s>arcgis.learn</s></code><s> and the model from scikit-learn followed by defining the model along with the parameters as follows:</s></p><p>First, we import the <code>MLModel</code> framework from <code>arcgis.learn</code>. Then, we specify the model to be used from scikit learn and define the necessary parameters as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>